packages/workflow-runtime/internal/index.ts

```diff
@@ -155,3 +157,12 @@ const passthroughOptions = (opts: ClientOptions): ClientOptions => {
 export const registerWorkflowRuntime = () => {
   registerRuntime("workflow", runtime);
 };
+
+const sensitiveLogKeys = ["airplane_token", "airplane_resources", "password", "token"];
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const logStringify = (obj: any): string => {
```

Comment:
nit since it doesn't do any logging. Does this need to be exported from the `index` file? If not, let's put it in `worker.ts` and then the interceptors file can import it. Let's try to keep unnecessary exports out of the `index.ts` file.

```suggestion
export const stringifyOmitSensitive = (obj: any): string => {
```

Reply:
Fixed.